Dockerfile

```
@@ -9,6 +9,20 @@
 
 FROM --platform=linux/amd64 python:3.12-slim-bookworm
 
+# OCI Labels 
+LABEL org.opencontainers.image.title="ScanCode Toolkit" 
+LABEL org.opencontainers.image.description="ScanCode Toolkit for code scanning and analysis." 
```

```suggestion
LABEL org.opencontainers.image.description="ScanCode detects licenses, copyrights, dependencies by scanning code to discover and inventory open source and third-party packages used in your code." 
```
Let's use our About section for our repo.